flow/util/utils.mk

```diff
@@ -109,8 +109,13 @@ ISSUE_CP_PLATFORM_FILE_VARS = LIB_FILES \
 ISSUE_CP_FILE_VARS = $(ISSUE_CP_DESIGN_FILE_VARS)
 ifndef EXCLUDE_PLATFORM
 ISSUE_CP_FILE_VARS += $(ISSUE_CP_PLATFORM_FILE_VARS)
+COUNT= $(shell ls -1 $(PLATFORM_DIR)/*.cfg 2>/dev/null | wc -l)
+ifeq ($(COUNT), 0)
+ISSUE_CP_FILES_PLATFORM = $(PLATFORM_DIR)/*.tcl
+else
```

Comment:
I think you could do
```
ISSUE_CP_FILES = $(wildcard $(PLATFORM_DIR)/*.tcl) $(wildcard $(PLATFORM_DIR)/*.cfg)
```
Which would be cleaner and pick up only files that exist

Reply:
I moved the logic in bash script as Vitor and I agreed, can you take a look?

Reply:
Whatever works. I think the only thing to be careful is that every variable needs to be `export`ed now, otherwise it won't get picked up by the script (I think).